core/src/main/java/com/google/instrumentation/common/Timestamp.java

```diff
@@ -90,6 +96,23 @@ public int getNanos() {
     return nanos;
   }
 
+  /**
+   * Returns the {@code Timestamp} calculated as this {@code Timestamp} plus {@code nanos}.
```

Comment:
Suggest: Returns a {@code Timestamp} calculated as this {@code Timestamp} plus some number of nanoseconds.